### ISSUE TITLE
Setting the webcam as the phone's camera 

### ISSUE THREAD

**Issue by hichemius** [vladikoff/chromeos-apk#189]
The apps that are based on cameras ( like Instagram for example ) crashes when I try to *"take a picture".*
Is there any way to make apps that rely on captures and pictures , **take their images from the webcam on a Linux computer**?

**Comment by rocketinventor** [comment]
Webcam apps (including Skype) work just fine for me so, it is probably just a problem with that app.

**Comment by hichemius** [comment]
yes but you didn't add anything ? for example the instagram app , when you click the camera button what shows up ?

**Comment by rocketinventor** [comment]
I didn't try packaging the Instagram app but, in other apps that would use
the built in camera on your phone, there is a camera app built into archon.
If you're on the beta version, then it looks like the stock camera app for
Android.

On Wed, Jul 1, 2015, 10:45 PM هشام العباسي - Hichem Abbassi <
notifications@github.com> wrote:

> yes but you didn't add anything ? for example the instagram app , when you
> click the camera button what shows up ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vladikoff/chromeos-apk/issues/189#issuecomment-117881769>
> .
>
-- 
Elliot Gerchak,
Owner of dementedLab.com
